.github/workflows/release.yml

```diff
@@ -24,24 +27,27 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
This comment is about line : 

    runs-on: ubuntu latest

It might be better to choose the version of ubuntu here, so that we decide when to switch to a newer ubuntu version instead of having to follow the rythm chosen by Github

Reply:
I've checked the labels here: https://github.com/actions/virtual-environments . We could choose `ubuntu-18.04` (which is what we've tested on), or else we could choose upcoming `ubuntu-20.04`, which is in preview. As we are still testing, we could test on this, and adopt it if it works.

Reply:
I agree, we could test ubuntu-20.04.
It will probably work as we're doing basic things, and would be a more recent baseline than 18.04

Reply:
Done.